diff -crB ./assets/imgui/backends/imgui_impl_glfw.cpp ./assets/imgui-cpy/backends/imgui_impl_glfw.cpp
*** ./assets/imgui/backends/imgui_impl_glfw.cpp	2024-08-14 11:27:05.234098638 -0300
--- ./assets/imgui-cpy/backends/imgui_impl_glfw.cpp	2024-08-14 11:24:18.274424018 -0300
***************
*** 73,79 ****
  //  2017-08-25: Inputs: MousePos set to -FLT_MAX,-FLT_MAX when mouse is unavailable/missing (instead of -1,-1).
  //  2016-10-15: Misc: Added a void* user_data parameter to Clipboard function handlers.
  
! #include "imgui.h"
  #ifndef IMGUI_DISABLE
  #include "imgui_impl_glfw.h"
  
--- 73,79 ----
  //  2017-08-25: Inputs: MousePos set to -FLT_MAX,-FLT_MAX when mouse is unavailable/missing (instead of -1,-1).
  //  2016-10-15: Misc: Added a void* user_data parameter to Clipboard function handlers.
  
! #include "../imgui.h"
  #ifndef IMGUI_DISABLE
  #include "imgui_impl_glfw.h"
  
diff -crB ./assets/imgui/backends/imgui_impl_glfw.h ./assets/imgui-cpy/backends/imgui_impl_glfw.h
*** ./assets/imgui/backends/imgui_impl_glfw.h	2024-08-14 11:27:05.234098638 -0300
--- ./assets/imgui-cpy/backends/imgui_impl_glfw.h	2024-08-14 11:23:01.616374504 -0300
***************
*** 18,24 ****
  // - Introduction, links and more at the top of imgui.cpp
  
  #pragma once
! #include "imgui.h"      // IMGUI_IMPL_API
  #ifndef IMGUI_DISABLE
  
  struct GLFWwindow;
--- 18,24 ----
  // - Introduction, links and more at the top of imgui.cpp
  
  #pragma once
! #include "../imgui.h"      // IMGUI_IMPL_API
  #ifndef IMGUI_DISABLE
  
  struct GLFWwindow;
diff -crB ./assets/imgui/backends/imgui_impl_opengl3.cpp ./assets/imgui-cpy/backends/imgui_impl_opengl3.cpp
*** ./assets/imgui/backends/imgui_impl_opengl3.cpp	2024-08-14 11:27:05.234098638 -0300
--- ./assets/imgui-cpy/backends/imgui_impl_opengl3.cpp	2024-08-14 11:24:33.251360709 -0300
***************
*** 112,118 ****
  #define _CRT_SECURE_NO_WARNINGS
  #endif
  
! #include "imgui.h"
  #ifndef IMGUI_DISABLE
  #include "imgui_impl_opengl3.h"
  #include <stdio.h>
--- 112,118 ----
  #define _CRT_SECURE_NO_WARNINGS
  #endif
  
! #include "../imgui.h"
  #ifndef IMGUI_DISABLE
  #include "imgui_impl_opengl3.h"
  #include <stdio.h>
diff -crB ./assets/imgui/backends/imgui_impl_opengl3.h ./assets/imgui-cpy/backends/imgui_impl_opengl3.h
*** ./assets/imgui/backends/imgui_impl_opengl3.h	2024-08-14 11:27:05.237432031 -0300
--- ./assets/imgui-cpy/backends/imgui_impl_opengl3.h	2024-08-14 11:24:08.780919497 -0300
***************
*** 26,32 ****
  //  Only override if your GL version doesn't handle this GLSL version. See GLSL version table at the top of imgui_impl_opengl3.cpp.
  
  #pragma once
! #include "imgui.h"      // IMGUI_IMPL_API
  #ifndef IMGUI_DISABLE
  
  // Follow "Getting Started" link and check examples/ folder to learn about using backends!
--- 26,32 ----
  //  Only override if your GL version doesn't handle this GLSL version. See GLSL version table at the top of imgui_impl_opengl3.cpp.
  
  #pragma once
! #include "../imgui.h"      // IMGUI_IMPL_API
  #ifndef IMGUI_DISABLE
  
  // Follow "Getting Started" link and check examples/ folder to learn about using backends!
Binary files ./assets/imgui/.git/index and ./assets/imgui-cpy/.git/index differ
diff -crB ./assets/imgui/.git/logs/HEAD ./assets/imgui-cpy/.git/logs/HEAD
*** ./assets/imgui/.git/logs/HEAD	2024-08-14 11:27:05.224098457 -0300
--- ./assets/imgui-cpy/.git/logs/HEAD	2024-08-14 11:20:33.503700893 -0300
***************
*** 1 ****
! 0000000000000000000000000000000000000000 29fadad1939ab62adcc5cd9339db7a784de90120 asari <32879431+deomorxsy@users.noreply.github.com> 1723645625 -0300	clone: from github.com:ocornut/imgui.git
--- 1 ----
! 0000000000000000000000000000000000000000 29fadad1939ab62adcc5cd9339db7a784de90120 asari <32879431+deomorxsy@users.noreply.github.com> 1723645233 -0300	clone: from github.com:ocornut/imgui.git
diff -crB ./assets/imgui/.git/logs/refs/heads/master ./assets/imgui-cpy/.git/logs/refs/heads/master
*** ./assets/imgui/.git/logs/refs/heads/master	2024-08-14 11:27:05.224098457 -0300
--- ./assets/imgui-cpy/.git/logs/refs/heads/master	2024-08-14 11:20:33.503700893 -0300
***************
*** 1 ****
! 0000000000000000000000000000000000000000 29fadad1939ab62adcc5cd9339db7a784de90120 asari <32879431+deomorxsy@users.noreply.github.com> 1723645625 -0300	clone: from github.com:ocornut/imgui.git
--- 1 ----
! 0000000000000000000000000000000000000000 29fadad1939ab62adcc5cd9339db7a784de90120 asari <32879431+deomorxsy@users.noreply.github.com> 1723645233 -0300	clone: from github.com:ocornut/imgui.git
diff -crB ./assets/imgui/.git/logs/refs/remotes/origin/HEAD ./assets/imgui-cpy/.git/logs/refs/remotes/origin/HEAD
*** ./assets/imgui/.git/logs/refs/remotes/origin/HEAD	2024-08-14 11:27:05.224098457 -0300
--- ./assets/imgui-cpy/.git/logs/refs/remotes/origin/HEAD	2024-08-14 11:20:33.503700893 -0300
***************
*** 1 ****
! 0000000000000000000000000000000000000000 29fadad1939ab62adcc5cd9339db7a784de90120 asari <32879431+deomorxsy@users.noreply.github.com> 1723645625 -0300	clone: from github.com:ocornut/imgui.git
--- 1 ----
! 0000000000000000000000000000000000000000 29fadad1939ab62adcc5cd9339db7a784de90120 asari <32879431+deomorxsy@users.noreply.github.com> 1723645233 -0300	clone: from github.com:ocornut/imgui.git
Only in ./assets/imgui-cpy/.git/objects/pack: pack-10d13656c65825ad3b82f5054935e2790c5d2204.idx
Only in ./assets/imgui-cpy/.git/objects/pack: pack-10d13656c65825ad3b82f5054935e2790c5d2204.pack
Only in ./assets/imgui-cpy/.git/objects/pack: pack-10d13656c65825ad3b82f5054935e2790c5d2204.rev
Only in ./assets/imgui/.git/objects/pack: pack-6810c8155e3fa9a7819c44bac93ea6c10cf6208e.idx
Only in ./assets/imgui/.git/objects/pack: pack-6810c8155e3fa9a7819c44bac93ea6c10cf6208e.pack
Only in ./assets/imgui/.git/objects/pack: pack-6810c8155e3fa9a7819c44bac93ea6c10cf6208e.rev
